packages/orchestrator-ui-components/src/rtk/api.ts

```diff
@@ -26,11 +38,17 @@ export const orchestratorApi = createApi({
             case BaseQueryTypes.fetch:
                 const fetchFn = fetchBaseQuery({
                     baseUrl: orchestratorApiBaseUrl,
+                    prepareHeaders: (headers) => {
```

Comment:
I don't think the function is necessary here and you can just do `prepareHeaders: prepareHeaders` instead.

Reply:
think so too and if key value have the same name `prepareHeaders,` is sufficient

Reply:
Done